README.md

```diff
@@ -88,7 +88,8 @@ How to execute plan with **params.json**: `bolt plan run autope --params @params
 This can also be used to deploy PE's large architecture without a fail over replica on AWS
 
 ```
-$(export-profile.py development)
+$python3 export-profile.py development
```

Comment:
Should this not be using the new aws_bastion_mfa_export.sh script ?

Reply:
Great catch!